api/pkg/cli/cmd/search/testdata/TestSearch_JSONFormat.golden

```diff
@@ -15,6 +15,7 @@
 				"ID": 12,
 				"Version": "0.2",
 				"DisplayName": "foo-bar-0.1",
+				"Deprecated": null,
```

Comment:
I think the default value should be `false`  ?
cc @pratap0007 @sm43

Reply:
for Non-deprecated resources deprecated field shouldn't include in the response

Reply:
I think,unspecified field gets nil pointer while unmarshaling hence there is nil